onetrain.py

```diff
@@ -18,16 +19,23 @@
 from rich.progress import Progress, TextColumn, BarColumn, TaskProgressColumn, TimeRemainingColumn, TimeElapsedColumn
 from tqdm import tqdm
 
+sys.path.append(os.path.abspath("/hello/hello@moodmagic.ai/onetrainer"))
```

Comment:
instead of hardcoded, this can be `os.environnent` call, so onetrainer path is defined in the shell script used to start onetrainer systemd service.

keep in mind that this wrapper is generic - i did write it for moodmagic, but there is nothing here moodmagic specific and it should not contain any moodmagic specific info.

Reply:
makes sense.  I will change this